### PR TITLE
Adds a one line stats line

### PR DESCRIPTION
**Pull request by ssbarnea**
Adds a one line starts at the end with information about number of found errors, warnings and files.

This adds one additional line with "Finished ..." on stderr that can be seen below: 
sample

stderrr is used instead of stdout in order to avoid affecting result that could be machine parseable. That also applies to any logging.

### CODE REVIEW

File: lib/ansiblelint/__main__.py [hunk 1]
```diff
@@ -84,24 +84,30 @@ def choose_formatter_factory(
...
+        if {match.rule.id, *match.rule.tags}.isdisjoint(options.warn_list):
+            failures += 1
+        else:
+            warnings += 1
+
     for id in sorted(matched_rules.keys()):
         if {id, *matched_rules[id].tags}.isdisjoint(options.warn_list):
             msg += f"  - '{id}'  # {matched_rules[id].shortdesc}\n"
-            failure = True
     for match in matches:
```
Comment (rafaelfolco) [inline lib/ansiblelint/__main__.py:111]: do we really need to iterate over matches again?

**Review by rafaelfolco — APPROVED** [review]
lgtm

File: lib/ansiblelint/__main__.py (hunk 1 above)
Comment (ssbarnea) [reply lib/ansiblelint/__main__.py:111]: That is not the right place to look for performance improvements. It is written as is for code readability, which is more important in this case.  Even in the worst case possible of having >1000 violations found, the cost is likely measures in few milliseconds.

Looking for performance is good, there are places where this really matters (see recent fix around parsing files multiple times, which made a 10x speed improvement).

**Review by greg-hellings — APPROVED** [review]

**Comment by ssbarnea** [comment]
Already merged.

**Pull request closed without merging**